src/odemis/driver/tescan.py

```diff
@@ -1127,12 +1160,25 @@ def _doMoveAbs(self, pos):
             logging.debug("Stage move to %s completed", pos)
             self._updatePosition()
 
-    def _doMoveRel(self, shift):
-        # Compute the position in absolute coordinates
+    def _doMoveRel(self, shift: dict):
+        """
+        Request the position in absolute coordinates and create new positions after applying shift values
+        :param shift: shift of linear axes in m or rotational axes in radians
```

Comment:
can you add what the keys and values are of this dict? I guess the axes names are the keys and the value is the corresponding shift.

Reply:
Indeed, typically you could specify the type as `Dict[str, float]`.

Reply:
Thanks I also added this specification to the _doMoveAbs method.